### PR TITLE
Merge install

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request adds the `UAGENT_MERGE_INSTALL` CMake option to allow merge installation, that is, that the library and its dependencies are installed with the same prefix (folder).

- [ ] Please, once approved https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/130, change `_microxrcedds_client_tag` to develop branch.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/151/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/145/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/153/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/147/)

File: ci/linux/CMakeLists.txt [hunk 1]
```diff
@@ -38,19 +38,19 @@ if(_have_c_fprofile_abs_path)
...
     INSTALL_DIR
-        ${PROJECT_BINARY_DIR}/temp_install
+        ${PROJECT_BINARY_DIR}/temp_install/isolated
     TEST_AFTER_INSTALL
         TRUE
     TEST_COMMAND
         COMMAND ${CMAKE_CTEST_COMMAND} -VV -T Test
-        COMMAND ${CMAKE_CTEST_COMMAND} -VV -T MemCheck -E "test-case*"
-        COMMAND ${CMAKE_CTEST_COMMAND} -VV -T Coverage -E "test-case*"
+#        COMMAND ${CMAKE_CTEST_COMMAND} -VV -T MemCheck -E "test-case*"
```
Comment (BorjaOuterelo) [inline ci/linux/CMakeLists.txt]: Why disable these tests?

Comment (julianbermudez) [reply ci/linux/CMakeLists.txt]: Enabled again.

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/154/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/148/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/155/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/149/)

**Pull request merged**